package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@adobe/aio-cli-plugin-api-mesh",
-	"version": "3.0.0",
+	"version": "3.0.1",
```

Comment:
Let's make sure that this version upgrade is not backward incompatible. Otherwise,  if it is a breaking change or a backward incompatible change, then either a major/minor bump would be needed